doc/man3/SSL_alloc_buffers.pod

```diff
@@ -0,0 +1,68 @@
+=pod
+
+=head1 NAME
+
+SSL_free_buffers, SSL_alloc_buffers - manage SSL structure buffers
+
+=head1 SYNOPSIS
+
+ #include <openssl/ssl.h>
+
+ int SSL_free_buffers(SSL *ssl);
+ int SSL_alloc_buffers(SSL *ssl);
+
+=head1 DESCRIPTION
+
+SSL_free_buffers() frees the read and write buffers of the given B<ssl>.
+SSL_alloc_buffers() allocates the read and write buffers of the given B<ssl>.
+Using these functions can save around 34k per SSL connection.
+
+The SSL_MODE_RELEASE_BUFFERS mode releases read or write buffers when the
+buffers have been drained. If this is too aggressive, the buffers can be
```

Comment:
"aggressive" is probably the most appropriate word, but it might not be understood by non-native speakers. Trying to figure out a better approach.